repo/branson/package.py

```diff
@@ -6,8 +6,10 @@
 from spack.package import *
 from spack.pkg.builtin.boost import Boost
```

Comment:
Referring to upstream packages changed in #953, but I think since `branson` was in the repo for that, you will get the appropriate update if merging develop into this.